### ISSUE TITLE
UI: repurpose /benchmarks view for benchmark-centric ("conceptual benchmark") overview

### ISSUE THREAD

**Issue by jgehrcke** [conbench/conbench#964]
Context: #787, #772

This view is not linked from the main/landing page:
![image](https://user-images.githubusercontent.com/265630/228260089-9e008bea-4c2f-4d63-864d-9f978f2ea23f.png)

Do we want to remove it? I think we want to use the `/benchmarks` path for showing the conceptual benchmarks, as discussed in #787.

**Comment by jonkeane** [comment]
Yeah, I'm totally fine repurposing it. Would it be possible | easy to copy this view over into. `/benchmarkResults` or `/benchmark-results`? Not totally sure if we need it inevitably or if we have anyone using it — but having an option to switch to that would be nice if it comes up (and slightly easier than resurrecting it out of git history).

And then: are we planning to also do the same in the API? IMO we probably should, though we will need to coordinate with a few folks | places in order to do so, since they currently rely in those endpoints (benchalerts is one place and also https://github.com/conbench/conbenchcoms). Both should be pretty easy + are in decently active development right now

**Comment by jgehrcke** [comment]
> Not totally sure if we need it inevitably or if we have anyone using it — but having an option to switch to that would be nice if it comes up

Let's keep it for now, did that quickly in #966.

> `/benchmark-results`?

This! :)

> And then: are we planning to also do the same in the API?

Wondered the same. Probably yes. Thanks for supporting the idea. I'd love to do that independently; I think there will be a time soonish when it feels like the right thing to do.

**Comment by jgehrcke** [comment]
Update: the 'conceptual benchmark' list/entrypoint is right now under a temporary URL, and will (I think, at leats) after all be served under benchmarks.

**Comment by austin3dickey** [comment]
That's good - I was confused what the "c-" meant until just now, looking at the title of this issue, when I realized it stands for "conceptual" :)
